src/main/scala/com/gu/support/config/StripeConfig.scala

```diff
@@ -1,12 +1,20 @@
 package com.gu.support.config
 
-import com.typesafe.config.Config
+import com.typesafe.config.{Config, ConfigException}
 
-case class StripeConfig(secretKey: String, publicKey: String) extends TouchpointConfig
+import scala.util.control.Exception.catching
 
-class StripeConfigProvider(config: Config, defaultStage: Stage, prefix: String = "stripe") extends TouchpointConfigProvider[StripeConfig](config, defaultStage) {
+case class StripeConfig(secretKey: String, publicKey: String, version: Option[String]) extends TouchpointConfig
+
+class StripeConfigProvider(config: Config, defaultStage: Stage, prefix: String = "stripe", version: Option[String] = None) extends TouchpointConfigProvider[StripeConfig](config, defaultStage) {
   def fromConfig(config: Config): StripeConfig = StripeConfig(
     secretKey = config.getString(s"$prefix.api.key.secret"),
-    publicKey = config.getString(s"$prefix.api.key.public")
+    publicKey = config.getString(s"$prefix.api.key.public"),
+    version = stripeVersion(config)
   )
+
+  def stripeVersion(config: Config): Option[String] = {
+    val catchMissing = catching(classOf[ConfigException.Missing])
```

Comment:
I think it might be better to use Config.hasPath() here as described in https://github.com/lightbend/config

Reply:
Yep, that's a much better idea! I've just made that change.